playbook/build_ami/main.yml

```diff
@@ -0,0 +1,184 @@
+---
```

Comment:
I'm guessing a lot of this is taken from AdHoc code on QPP that you already know works, but I still have the same question about whether you could do this with Packer.  Not the most high priority thing, but I suspect it would make this a bit easier/less custom (this file seems to mostly reimplement the Packer steps).

Reply:
I would like to use packer for this, but I'm not sure how to write a packer template that works properly given all the references to various roles in this playbook. If I have time, I'll experiment with writing up a template.